code/mouse.py

```diff
@@ -181,6 +181,7 @@ def mouse_scroll_down():
     def mouse_scroll_down_continuous():
         """Scrolls down continuously"""
         global continuous_scoll_mode
+        stop_scroll()
```

Comment:
I believe this will break the ability to speed up the continuous scroll via repeaters as-is, no?

This should probably be fenced somehow, maybe via the `continuous_scoll_mode` variable or something, though it probably needs to be reset properly in stop_scroll

Reply:
No, speeding up is still possible using "wheel up", "wheel down" as this does not pass via the mouse_scroll_down/up_continuous.

Reply:
Or is "wheel downer" also supposed to speed up scrolling?

Reply:
I have updated the PR to allow speeding up with both "wheel down" and "wheel downer"